src/main/java/controller/MainController.java

```diff
@@ -0,0 +1,108 @@
+package controller;
+
+import domain.AbsenceHistory;
+import domain.Attendance;
+import domain.AttendanceState;
+import domain.Calender;
+import domain.Crew;
+import dto.AbsenceResultDto;
+import dto.AttendanceResultDto;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import util.FileManager;
+import view.InputView;
+import view.OutputView;
+
+public class MainController {
+
+    private LocalDate today;
+    private int todayMonth;
+    private int todayDay;
+    private String todayDayOfWeek;
+    private Attendance attendance;
+
+    public void run() {
+        prepareToday();
+        String feature;
+        do {
+            feature = InputView.inputFeature(todayMonth, todayDay, todayDayOfWeek);
+            if (feature.equals("1")) {
```

Comment:
```suggestion
            if ("1".equals(feature)) {
```
문자열에 대한 검증을 할 때는 상수인 문자열에 대한 equals 를 호출해주시는게 좋아요~
feature.equals() 를 했을 때 feature 가 만약 null 인 경우에는 equals 호출 시 NPE 가 발생할거에요